<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveInfoDialog</class>
 <widget class="QDialog" name="SaveInfoDialog">
  <property name="windowTitle">
   <string>File Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="bannerImg">
     <property name="minimumSize">
      <size>
       <width>192</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>192</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;NO BANNER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Region</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="ntscURB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>NTSC-U</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">regionBtnGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ntscKRB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>NTSC-K</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">regionBtnGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ntscJRB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>NTSC-J</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">regionBtnGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="palRB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>PAL</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">regionBtnGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QLabel" name="checkSumLbl">
     <property name="text">
      <string>Checksum:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QLabel" name="adventureCountLbl">
     <property name="text">
      <string>Adventure Count:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="currentAdventureLbl">
     <property name="text">
      <string>Current Adventure:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="titleLbl">
     <property name="text">
      <string>{TITLE}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="subtitleLbl">
     <property name="text">
      <string>{SUBTITLE}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regionBtnGroup</sender>
   <signal>buttonToggled(QAbstractButton*,bool)</signal>
   <receiver>SaveInfoDialog</receiver>
   <slot>onRegionChanged(QAbstractButton*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRegionChanged(QAbstractButton*,bool)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="regionBtnGroup"/>
 </buttongroups>
</ui>
